src/components/Overlay/Overlay.md

```diff
@@ -0,0 +1,24 @@
+# Overlay
+Partially obscures the page. Typically used with a Dialog or Panel.
```

Comment:
I would add - "Partially obscures the page to create a modal experience and focus the user's attention."